hazelcast/generated-sources/src/hazelcast/client/protocol/codec/codecs.h

```diff
@@ -83,13 +103,21 @@ struct HAZELCAST_API client_addclusterviewlistener_handler
       int32_t version,
       std::vector<std::pair<boost::uuids::uuid, std::vector<int>>> const&
         partitions) = 0;
-};
 
-/**
-                 * Triggers partition assignment manually on the cluster.
-                 * Note that Partition based operations triggers this automatically
- */
-ClientMessage HAZELCAST_API client_triggerpartitionassignment_encode();
+    /**
+     * @param version Holds the state of member-groups and member-list-version
+     * @param member_groups Grouped members by their UUID. Grouping is done
+     * based on RoutingStrategy.
+     */
+    virtual void handle_membergroupsview(
+      int32_t version,
+      std::vector<std::vector<boost::uuids::uuid>> const& member_groups) = 0;
+
+    /**
+     * @param version The cluster version.
+     */
+    virtual void handle_clusterversion(internal::version const& version) = 0;
```

Comment:
```suggestion
    virtual void handle_clusterversion(version const& version) = 0;
```

When I regenerated the `client-protocol`, I got a (_very slightly_) different output.

I'm not _really_ saying we should change it. Just trying to understand if it's outdated, or our tooling differs.

Reply:
I think you did not update the client protocol repo to latest. See https://github.com/hazelcast/hazelcast-client-protocol/blob/41a96a9bb2798bfff174c52140781d7eb36d2bae/cpp/__init__.py#L127 line.

Reply:
Ok, there was a duplication which I fixed now at [this](https://github.com/hazelcast/hazelcast-client-protocol/pull/558) PR. Should be fine after this PR.